converters/fromZigbee.js

```diff
@@ -1107,6 +1107,17 @@ const converters = {
             };
         },
     },
+    smartsense_multi: {
```

Comment:
I needed this because ias_contact_status_change  doesn't work here:

that has
msg.data.zoneStatus

whereas I need 
msg.data.data.zoneStatus

Rather than making a fork, is it likely that it's a bug in the existing ias_contact_status_change?

ias_zone_motion_dev_change  uses data.data the other 3 ias convertors do not.